tests/boot/test_mcuboot/testcase.yaml

```diff
@@ -0,0 +1,17 @@
+common:
+    sysbuild: True
+    harness: console
+    harness_config:
+      type: one_line
```

Comment:
It an error, should be `type: multi_line`. In the current state the test only looks for the first line and passes just after the first flash.

Reply:
Thanks for the heads up, I will resolve this on next push.